nodejs/week2/homework/meal-sharing/src/backend/api/meals-router.js

```diff
@@ -0,0 +1,65 @@
+const express = require("express");
+const router = express.Router();
+
+const meals = require("./../data/meals.json");
+
+router.get("/", async (request, response) => {
+  try {
+    console.log(meals);
+    console.log("in /api/meals");
+
+    const maxPrice = request.query.maxPrice;
+    const titles = request.query.title;
+    const createdAfter = request.query.createdAfter;
+    const limit = request.query.limit;
+    if (maxPrice !== undefined && maxPrice !== "") {
+      console.log("maxPrice");
+      const filteredMeals = meals.filter((meal) => meal.price < maxPrice);
+      response.send(filteredMeals);
+    } else if (titles !== undefined && titles !== "") {
+      console.log("titles");
+      const filteredTitle = meals.filter((meal) => meal.title.includes(titles));
```

Comment:
a good practice to have in mind while working/comparing string variables is that one could have lowercase characters and another uppercase, for instance 'Indian food' vs 'indian food' are same right?
Could you do something to make them both either lowercase or uppercase so the comparison works for that case as well?